GitHubA11yFixes.user.js

```diff
@@ -15,6 +15,24 @@ function makeHeading(elem, level) {
 	elem.setAttribute("aria-level", level);
 }
 
+var idCounter = 0;
+// Get a node's id. If it doesn't have one, make and set one first.
+function setAriaIdIfNecessary(elem) {

```

Comment:
I'd prefer this were renamed to `getAriaId` or similar. I know it "sets" as well in some cases, but from the caller's perspective, it's a getter.